types/src/transaction.rs

```diff
@@ -308,13 +383,17 @@ impl CanonicalDeserialize for TransactionPayload {
 enum TransactionPayloadType {
     Program = 0,
     WriteSet = 1,
+    Script = 2,
```

Comment:
not sure what that would mean in terms of work to do in this diff. 
If there is any work can you point me to something to look at?
Otherwise please let me know that is not requiring changes. 
Thanks!

Reply:
I don't think there's an easy solution right now. Let's revisit this issue later